src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt

```diff
@@ -7,80 +7,138 @@ import me.zero.alpine.listener.EventHook
 import me.zero.alpine.listener.Listener
 import me.zeroeightsix.kami.event.events.PacketEvent
 import me.zeroeightsix.kami.module.Module
+import me.zeroeightsix.kami.setting.Setting
 import me.zeroeightsix.kami.setting.Settings
-import me.zeroeightsix.kami.util.math.MathUtils.reverseNumber
+import me.zeroeightsix.kami.util.TimerUtils
 import me.zeroeightsix.kami.util.text.MessageSendHelper.sendServerMessage
-import net.minecraft.network.play.client.CPacketAnimation
 import net.minecraft.network.play.server.SPacketChat
 import net.minecraft.util.EnumHand
-import java.util.*
+import net.minecraft.util.math.BlockPos
+import net.minecraftforge.fml.common.gameevent.InputEvent
+import net.minecraftforge.fml.common.gameevent.InputEvent.KeyInputEvent
+import kotlin.math.max
+import kotlin.random.Random
 
 /**
  * Created by 086 on 16/12/2017.
  * Updated by dominikaaaa on 21/04/20
+ * Updated by Xiaro on 09/09/20
+ *
  * TODO: Path finding to stay inside 1 chunk
  * TODO: Render which chunk is selected
  */
-@Module.Info(name = "AntiAFK", category = Module.Category.MISC, description = "Prevents being kicked for AFK")
+@Module.Info(
+        name = "AntiAFK",
+        category = Module.Category.MISC,
+        description = "Prevents being kicked for AFK"
+)
 class AntiAFK : Module() {
-    private val frequency = register(Settings.integerBuilder("ActionFrequency").withMinimum(1).withMaximum(100).withValue(40).build())
-    val autoReply = register(Settings.b("AutoReply", true))
-    private val swing = register(Settings.b("Swing", true))
-    private val jump = register(Settings.b("Jump", true))
+    private val delay = register(Settings.integerBuilder("ActionDelay").withValue(50).withRange(0, 100).build())
+    private val variation = register(Settings.integerBuilder("Variation").withValue(25).withRange(0, 50))
     private val squareWalk = register(Settings.b("SquareWalk", true))
```

Comment:
Make this off by default, and rename it to "Walk" or something

Also, disabling it should cancel Baritone